PAS_Database/dbo/Stored Procedures/Procs1/GetBulkStockLineAccountingDetailsViewById.sql

```diff
@@ -21,17 +22,21 @@ BEGIN
  SET NOCOUNT ON;    
  BEGIN TRY        
    BEGIN  
-     DECLARE @blkSTKModuleID INT = 2; 
-				DECLARE @ManagementStructureModuleId BIGINT = 0;   
+				DECLARE @blkSTKModuleID INT = 2; 
+				DECLARE @ManagementStructureModuleId BIGINT = 0; 
+				DECLARE @JournalBatchDetailId BIGINT = 0;
+
+				SELECT TOP 1 @JournalBatchDetailId = JournalBatchDetailId FROM [dbo].[BulkStocklineAdjPaymentBatchDetails] WITH(NOLOCK) WHERE ReferenceId = @ReferenceId;
```

Comment:
Is it correct to take TOP 1 ?

Reply:
Yes, Every time value same for all, but we take 1 for process so.